### PR TITLE
Implement minimum player counts

### LINKED ISSUE #6538: Set minimum players when creating game
When creating a game, you can set the maximum number of players but not the minimum. Thus, if you only want to play a five player game, it will be advertised as 3-5 players.

And a player who wants to join a game with exactly five players might end up in three player game.

It would be nice if players could set both the minimum and maximum number of players when creating games.

### PR DESCRIPTION
**Pull request by crericha**
Fixes #6538

### CODE REVIEW

**Comment by crericha** [comment]
@roseundy, please verify that using max player count for 1825 unit selection and incorporating min player count into filtered rules meets your intentions.

File: assets/app/view/create_game.rb [hunk 1]
```diff
@@ -342,7 +358,7 @@ def mode_selector
 
     def mode_input(mode, text)
       click_handler = lambda do
-        store(:mode, mode, skip: true)
+        store(:mode, mode) # , skip: true)
```
Comment (tobymao) [inline assets/app/view/create_game.rb:361]: why was this changed? should you remove the comment

**Review by tobymao — APPROVED** [review]

File: assets/app/view/create_game.rb (hunk 1 above)
Comment (crericha) [reply assets/app/view/create_game.rb:361]: To hide the minimum players input when hotseat is selected. I will verify it is still needed and if so remove the commented out portion.

Comment (crericha) [reply assets/app/view/create_game.rb]: Not needed. Reverted.
